gm4_sweethearts/data/sweethearts/functions/init.mcfunction

```diff
@@ -0,0 +1,19 @@
+#announce module installation
+tellraw @a[gamemode=creative] ["",{"text":"[GM4]: Installing Sweethearts..."}]
+execute unless entity @p run say GM4: Installing Sweethearts...
+
+#declare and initialise scoreboards and settings
+scoreboard players set updateHappened gm4_upCheck 1
+scoreboard players set sweethearts gm4_modules 1
+scoreboard players set sweethearts gm4_clockTick 0
+
+scoreboard objectives add gm4_sh_sneak minecraft.custom:minecraft.sneak_time
+scoreboard objectives add gm4_sh_health health
```

Comment:
Health scoreboards are immutable, so having overlapping objectives names is okay. gm4_health is preferred here.